src/components/Weather/Weather.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { WeatherBody } from "./WeatherBody";
+import "./WeatherBody.css";
+
+const Weather = () => {
+  const [lat, setLat] = useState([]);
+  const [long, setLong] = useState([]);
```

Comment:
Data type of lat and long as array seems to be not required here  can use string too.